### DISCUSSION TITLE
Multi-layer sea bottom

### DISCUSSION THREAD

**Discussion by daka-uiuc** [org-arl/UnderwaterAcoustics.jl#51] [Q&A]
How can I implement a multi-layer seabed like that of the SWellEx-96 experiment, using Kraken?

**Comment by mchitre** [comment]
Which propagation model are you working with?

**Reply by daka-uiuc** [reply]
I am using Kraken.

**Reply by mchitre** [reply]
Currently we don't have a way of adding multilayer seabeds. In principle, we can add that support by defining an appropriate `seabed` model, but it may need some careful thinking through. Do you have example Kraken env files for multilayer seabeds that we can use as a starting point?

**Reply by daka-uiuc** [reply]
Thanks for the response. Unfortunately, I do not have any Kraken env files.

**Reply by mchitre** [reply]
Added an issue to add support for this:
https://github.com/org-arl/UnderwaterAcoustics.jl/issues/52

Will implement when I get a chance. Alternatively, if anyone wants to take this up and raise a PR, I am open to it.

**Comment by mchitre** [comment]
To summarize an answer...

We currently don't have support for multiple seabed layers in the environment description. We will need to add that support (https://github.com/org-arl/UnderwaterAcoustics.jl/issues/52) and also in models like Kraken from [UnderwaterAcoustics.jl](https://github.com/org-arl/UnderwaterAcoustics.jl).

To be added soon.
